independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeployment.java

```diff
@@ -123,12 +125,9 @@
         this.interceptorBindings = findInterceptorBindings(index);
         for (InterceptorBindingRegistrar registrar : bindingRegistrars) {
             for (DotName dotName : registrar.registerAdditionalBindings()) {
-                ClassInfo classInfo = index.getClassByName(dotName);
+                ClassInfo classInfo = getClassByName(index, dotName);
```

Comment:
Shouldn't we use the logging variant here? I mean we threw runtime exception before and now silently ignore the problem?

Reply:
This is the logging variant.

But yes, we threw exception before, but since we will now have logging, in this case I reconsidered and thought that should be enough instead of hard fail. WDYT?

Reply:
Ah sorry, these Fridays...